assets/pages/regular-contributions/components/emailConfirmation.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import PageSection from 'components/pageSection/pageSection';
+
+
+// ----- Component ----- //
+
+export default function EmailConfirmation() {
+
+  return (
+    <div className="component-email-confirmation">
+      <PageSection modifierClass="email-confirmation">
+        <p className="component-email-confirmation__copy">
+          Look out for an email confirming your recurring payment.
+        </p>
+      </PageSection>
+    </div>
+  );
+
+}
```

Comment:
Might be worth sticking this in the shared components folder, it doesn't rely on anything in the page and it could be re-usable at some point.

Reply:
I think it's better to keep it here where it's in context, I really can't see it ever being used anywhere else, but if it is we can move it at that point.